CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.1.0]
+### Changed
```

Comment:
If we're bumping this version to 8.1.0 (and I think that's right), should this be "Added"?

``` suggestion
### Added
```

Reply:
Okay, I can see that. Well, you're also bumping the version from 8.0.0 to 8.1.0, so does it make sense to bump to just 8.0.1?

Reply:
I guess my thought was that you bump the minor to communicate to consumers that there's something new in the package they might want to know about. And so if that's the case, the new thing should be listed under Added. But I guess there's room for interpretation. Not a big deal.